runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineWorkerPoolOptions.java

```diff
@@ -204,18 +204,20 @@ public String create(PipelineOptions options) {
   List<String> getFilesToStage();
   void setFilesToStage(List<String> value);
 
+  String getWorkerDiskType();
+
   /**
-   * Specifies what type of persistent disk should be used. The value should be a full or partial
-   * URL of a disk type resource, e.g., zones/us-central1-f/disks/pd-standard. For
-   * more information, see the
-   * <a href="https://cloud.google.com/compute/docs/reference/latest/diskTypes">API reference
-   * documentation for DiskTypes</a>.
+   * Specifies what type of persistent disk should be used. The value should be a
+   * full disk type resource, e.g.,
+   * compute.googleapis.com/projects//zones//diskTypes/pd-ssd. For more information,
```

Comment:
this is directly from this documentation: https://cloud.google.com/dataflow/pipelines/troubleshooting-your-pipeline under 'Your job is shuffle-bound'